### ISSUE TITLE
Feat:Add Client Testimonials and FAQ page to the project.

### ISSUE THREAD

**Issue by Pratik2401** [XceeDesigns/Website#58]
### Describe the feature

Introduction:

Briefly welcome visitors and explain the page's purpose.
Set the expectation that they'll find client testimonials and frequently asked questions here.
Testimonials:

Showcase client testimonials with accompanying visuals.
Include clients' names and endorsements for authenticity.
FAQ:

Organize questions into categories for easy navigation.
Provide clear and concise answers to common inquiries.
Call-to-Action:

Prompt visitors to take action with a clear call-to-action button.
Encourage them to contact you or explore your services further.

### Add ScreenShots

.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Pratik2401** [comment]
pls assign me these under GSSOC'24 @XceeDesigns

**Comment by XceeDesigns** [comment]
@Pratik2401 Any update ?

**Comment by Pratik2401** [comment]
Yes almost done I will request pr till tomorrow evening sorry for delay but my endsem are going on

**Comment by XceeDesigns** [comment]
@Pratik2401 take your time.
Just want to know the update.

Happy Coding

**Comment by Pratik2401** [comment]
Hey, should I make testimonial and FAQ page in a same or different page or I can chaneg that before PR

**Comment by XceeDesigns** [comment]
Add testimonial on landing page and create a seperate page for FAQ.

**Comment by Pratik2401** [comment]
![image](https://github.com/XceeDesigns/Website/assets/105542579/4d78c3fd-8a27-4018-a057-a1cd1a252ecf)

**Comment by Pratik2401** [comment]
![image](https://github.com/XceeDesigns/Website/assets/105542579/960c2756-73b4-43eb-8dc7-d692f7bc662b)

**Comment by Pratik2401** [comment]
heyya! Can you review and merge the request
